client/src/host/HostMain.js

```diff
@@ -5,12 +5,10 @@ import './components/HostControls'
 let HostRequestor = {
     clearSessionsP: () => StaticRequestor.getP(`/sessions/clear`),
     createSessionP: () => StaticRequestor.postP(`/sessions/create`),
-    createP: (sessionId, playerIds, deck) => StaticRequestor.postP(`/sessions/${sessionId}/game/${JSON.stringify(playerIds)}/${JSON.stringify(deck)}`),
+    createP: (sessionId, deck) => StaticRequestor.postP(`/sessions/${sessionId}/game/`, {deck: deck}),
```

Comment:
The trailing "/" is a different route than no trailing "/". Would be preferable to not include it. Treat the game as "/game" and not "/game/".